packages/vow/src/vow.js

```diff
@@ -33,14 +37,18 @@ export const prepareVowKit = (zone, vowRejectionTracker) => {
    * Get the current incarnation's promise kit associated with a vowV0.
    *
    * @param {VowResolver} resolver
+   * @param {{ potentiallyHandled?: boolean }} [options]
    */
-  const provideCurrentKit = resolver => {
+  const provideCurrentKit = (resolver, options) => {
     let pk = resolverToEphemera.get(resolver);
```

Comment:
Nit. This is no longer just a `pk`, and the existing value may contain all possible properties of the ephemera, let's name it as such.

Reply:
Done.